### ISSUE TITLE
Issue with `performCount` on Ember v3.15

### ISSUE THREAD

**Issue by Turbo87** [machty/ember-concurrency#337]
We tried to update our app to Ember 3.15 this morning and the test suite of one of our apps uncovered the following issue:

> Assertion Failed: You attempted to update `performCount` on `<(unknown):ember22520>`, but it had already been used previously in the same computation.  Attempting to update a value after using it in a computation can cause logical errors, infinite revalidation bugs, and performance issues, and is not supported.
>
> `performCount` was first used:
>
> - While rendering:
>     (instance of a `(unknown function)` modifier)
> 
> Stack trace for the first usage: 
>     at get (http://localhost:7357/assets/vendor.js:39430:11)
>     at Class.incrementProperty (http://localhost:7357/assets/vendor.js:53987:72)
>     at Class.schedule (http://localhost:7357/assets/vendor.js:158913:12)
>     at Class._performShared (http://localhost:7357/assets/vendor.js:160316:23)
>     at Class.perform (http://localhost:7357/assets/vendor.js:160286:19)
>     at XValidatedInput._focus (http://localhost:7357/assets/vendor.js:143177:33)
>     at XValidatedInput.focusIn (http://localhost:7357/assets/vendor.js:143157:12)
>     at XValidatedInput.trigger (http://localhost:7357/assets/vendor.js:59364:23)

I haven't been able to come up with a reproduction yet, but I'll update this issue if I can provide one.

**Comment by maxfierke** [comment]
Seems to be triggered by this: https://github.com/machty/ember-concurrency/blob/a863f4172fca5a63104054ff7a7aa41796a41380/addon/-scheduler.js#L54

Mind giving this branch a try: https://github.com/machty/ember-concurrency/pull/338 ?

**Comment by robclancy** [comment]
I been seeing this issue too.

**Comment by robclancy** [comment]
That branch fixes it, I have to use it now or can't release.

**Comment by maxfierke** [comment]
I'm having issues with the test for the fix for it on earlier Ember versions in the CI matrix, but I'll try to fix that up tonight and tag a release. I do not recommend using a branch in production.

**Comment by Turbo87** [comment]
@maxfierke it might be possible to remove the test file from the broccoli tree depending on the ember version, but I have to admit that I'm not sure how exactly to do that either 😅

**Comment by robclancy** [comment]
I mean.. I can't not use the branch it literally breaks everything.

**Comment by Turbo87** [comment]
that double negation is hard to parse 😅

**Comment by allthesignals** [comment]
I'm seeing something similar:
![image](https://user-images.githubusercontent.com/5004319/73200903-bf660e80-4105-11ea-875b-279ddd76f89b.png)

It's on "numQueued".

Versions:
```
    "ember-concurrency": "^1.1.5",
    "ember-concurrency-decorators": "^1.0.0",
```
code:
```javascript
export default class FindModalSplit extends Helper {
  /**
   * Task that wraps async call to get modal-split record from store
   */
  @task *findModalSplitTask(geoid) {
    return yield [
      this.readonlyStore.find('ACS-modal-split', geoid),
      this.readonlyStore.find('CTPP-modal-split', geoid),
    ];
  }

  /**
   * Main helper 'compute' function
   */
  compute(params) {
    const [geoid] = params;
    if (!geoid) return null;
    return this.findModalSplitTask.perform(geoid);
  }
}
```

**Comment by maxfierke** [comment]
@allthesignals can you confirm there are no addons depending on earlier versions of ember-concurrency that might be getting used instead?

**Comment by allthesignals** [comment]
@maxfierke that was totally it! time to add https://github.com/salsify/ember-cli-dependency-lint to this project... 

appreciate it :)
